butte-build/src/ast/parser.rs

```diff
@@ -14,12 +14,12 @@ use nom::{
     sequence::*, IResult,
 };
 
-use std::{path::Path, str::FromStr};
+use std::{iter::FromIterator, path::Path, str::FromStr};
 
 #[cfg(test)]
 macro_rules! assert_failed_parse {
     ($left:expr, $rest:expr, $error_kind:ident) => {
-        assert_eq!(
+        pretty_assertions::assert_eq!(
```

Comment:
How about a:
```rust
#[cfg(test)]
use pretty_assertions::assert_eq;
```
at the top of the module instead of giving the full path here?

Reply:
Yup.

Reply:
Ah actually `assert_eq` clashes with the Rust prelude `assert_eq` (when using `super::*`). I can use `self::assert_eq` to resolve, but that doesn't really help much. Looks like we can't just drop it in. I've already removed the `super::*` in tests since it discovered a bug in `parser_macros.rs` so I'll push that change up.